launch/dira.launch

```diff
@@ -0,0 +1,34 @@
+<launch>
+    <!-- Common settings -->
+    <arg name="rgb_topic" default="/camera/rgb/image/compressed" />
+    <arg name="cmd_topic" default="/cmd_vel" />
+    <arg name="queue_size" default="10" />
+    <arg name="is_show_image" default="true" />
+
+    <!-- This argument is fixed -->
+    <arg name="adapter" value="dira" />
+
+    <!-- DiRA's specific settings -->
+    <arg name="team" default="team220" />
+    <arg name="address" default="127.0.0.1" />
+    <arg name="port" default="9090" />
+
+    <!-- Start rosbridge_server -->
+    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">
+        <arg name="address" value="$(arg address)" />
+        <arg name="port" value="$(arg port)" />
+    </include>
```

Comment:
I realize that we should add `rosbridge_server` as a required package in README.

Reply:
Yeah, I agree with that.